typings/index.d.ts

```diff
@@ -1,6 +1,8 @@
 // Silence errors from libraries with no type defintions provided
 declare module 'intersection-observer';
 declare module 'object-fit-images';
+// TODO(giles): remove once we add types to tp-icons
+declare module '@thumbtack/thumbprint-icons';
```

Comment:
Why is this needed if `icon` is `React.ReactNode`?

Reply:
it's imported by the test suite, which throws an error right now

Reply:
Ah, ok. Makes sense!